### PR TITLE
Switch to using requester pays on storage buckets

### PR DESCRIPTION
**Pull request by zaneselvans**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -50,55 +50,89 @@ Currently available datasets
...
-See the `issues in this repository
+See the `issues in the pudl-catalog repository
 <https://github.com/catalyst-cooperative/pudl-catalog/issues>`__ for more
 detailed tasks.
 
-Planned data distribution system
+Usage
+-----
+
+Public data and "requester pays"
```
Comment (cmgosnell) [inline README.rst:74]: can we add this stuff into the data access page instead of the readme??

File: README.rst [hunk 2]
```diff
@@ -50,55 +50,89 @@ Currently available datasets
...
+  in initial cloud credits.
+* `Create a Cloud Billing account <https://cloud.google.com/billing/docs/how-to/manage-billing-account#create_a_new_billing_account>`__
+  associated with the project and `enable billing for the project
+  <https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project>`__
+  through this account.
+* `Using Google Cloud IAM <https://cloud.google.com/iam/docs/granting-changing-revoking-access#granting-console>`__,
+  add the **Service Usage Consumer** role to your account, which enables it to make
+  billed requests on the behalf of the project.
+* Install the `gcloud utilities <https://cloud.google.com/sdk/docs/install>`__ on your
+  computer. This can be done using ``conda`` (or ``mamba``):
```
Comment (cmgosnell) [inline README.rst:107]: I would suggest this link: https://anaconda.org/conda-forge/google-cloud-sdk

File: README.rst [hunk 3]
```diff
@@ -50,55 +50,89 @@ Currently available datasets
...
+  conda install -c conda-forge google-cloud-sdk
+
+* Initialize the ``gcloud`` command line interface, logging into the account used to
+  create the aforementioned project and selecting it as the default project; this will
+  allow the project to be used for requester pays access through the command line:
+
+.. code:: bash
+
+  gcloud auth login
+  gcloud init
```
Comment (cmgosnell) [inline README.rst:120]: this step asked me some questions:

```
Pick configuration to use:
 [1] Re-initialize this configuration [default] with new settings 
 [2] Create a new configuration
```

**Review by cmgosnell — COMMENTED** [review]
some questions....

File: README.rst (hunk 3 above)
Comment (cmgosnell) [reply README.rst:120]: it also asked:
```
Pick cloud project to use: 
 [1] northern-hope-350719
 [2] pudl-test
 [3] Enter a project ID
 [4] Create a new project
```

(I had just made the `pudl-test` project when i first logged in)

File: README.rst [hunk 4]
```diff
@@ -50,55 +50,89 @@ Currently available datasets
...
-Example Usage
--------------
+* To test whether your GCP account is set up correctly and authenticated you can run the
+  following command to list the contents of the cloud storage bucket containing the
+  intake catalog data:
 
-See the notebook included in this repository for more details.
+.. code:: bash
+
+   gsutil ls gs://intake.catalyst.coop
```
Comment (cmgosnell) [inline README.rst:135]: okay i got:

```
(pudl-dev) %0 ~/code% gsutil ls gs://intake.catalyst.coop                                                            christinagosnell@New-Shiny-Thing
BadRequestException: 400 Bucket is a requester pays bucket but no user project provided.
[1]    18496 exit 1     gsutil ls gs://intake.catalyst.coop
``

File: README.rst (hunk 3 above)
Comment (cmgosnell) [reply README.rst:120]: i choose 2

Comment (zaneselvans) [reply README.rst:120]: Which one did you choose? I think it should be the project you just created, and within which you should have given yourself the permission to spend money.

File: README.rst (hunk 4 above)
Comment (zaneselvans) [reply README.rst:135]: What do you get when you do
```sh
gcloud config configurations list
```

Comment (zaneselvans) [reply README.rst:135]: (but good to know that the `gsutil ls` is a sufficient check!)

Comment (cmgosnell) [reply README.rst:135]:
```
(pudl-dev) %0 ~/code% gcloud config configurations list                                                              christinagosnell@New-Shiny-Thing
NAME     IS_ACTIVE  ACCOUNT               PROJECT    COMPUTE_DEFAULT_ZONE  COMPUTE_DEFAULT_REGION
default  True       gosnellc11@gmail.com  pudl-test

**Review by cmgosnell — COMMENTED** [review]
for me, the most confusing part of this was the initial setup of the account and project. It felt very fuck-around-and-find-out.... some lil screenshots would be helpful. 

(a part of my problem was i was trying to do this on my non-primary google account so I could read the instructions on all of the links but not actually do anything there.)

and i commented on the several command line hurdles

File: README.rst (hunk 1 above)
Comment (zaneselvans) [reply README.rst:74]: I'll break it out into a separate page and link to it. I agree this is too much detail for the README.

File: README.rst (hunk 2 above)
Comment (zaneselvans) [reply README.rst:107]: The command immediately following this is enough right?

**Pull request merged**